src/commands/staff/RemovePunishmentHistory.ts

```diff
@@ -0,0 +1,94 @@
+import {ArgumentType, BaseCommand, ICommandContext, ICommandInfo} from "../BaseCommand";
```

Comment:
I'm not sure what the deal is with this command.  Either it is misnamed, or misimplemented.

removepunishment should take a moderation id and remove a single punishment
removepunishmenthist should take a NAME and remove ALL their punishments.  That's what a history is.

Also typo under cmdCode

Reply:
It should be the former. We're just removing one entry from the punishment history, not every entry.